sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -744,40 +751,54 @@ number
     | MINUS? BIGDECIMAL_LITERAL       #bigDecimalLiteral
     ;
 
+ansiReserved
+    // A list of reserved keywords in SparkSQL. These keywords are reserved in all the ANSI SQL standards
+    // (SQL-92, SQL-99, SQL-2003, SQL-2008, SQL-2011, and SQL-2016) and PostgreSQL.
+    : ALL | AND | ANY | AS | AUTHORIZATION | BOTH | CASE | CAST | CHECK | COLLATE | COLUMN | CONSTRAINT | CREATE
+    | CROSS | CURRENT_DATE | CURRENT_TIME | CURRENT_TIMESTAMP | CURRENT_USER | DISTINCT | ELSE | END | EXCEPT
+    | FALSE | FETCH | FOR | FOREIGN | FROM | FULL | GRANT | GROUP | HAVING | IN | INNER | INTERSECT | INTO | IS
+    | JOIN | LEADING | LEFT | NATURAL | NOT | NULL | ON | ONLY | OR | ORDER | OUTER | OVERLAPS | PRIMARY
+    | REFERENCES | RIGHT | SELECT | SESSION_USER | SOME | TABLE | THEN | TO | TRAILING | UNION | UNIQUE | USER
+    | USING | VARYING | WHEN | WHERE | WITH
```

Comment:
In PostgreSQL, VARYING is non-reserved. Thus, could we change it to non-reserved first?

Reply:
ok